ansible/ansible.cfg

```diff
@@ -0,0 +1,9 @@
+[defaults]
+# Include 3rd party roles directory as well as local custom roles.
+roles_path = roles:galaxy-roles
+# Disable retry files.
+retry_files_enabled = False
+
+# Enable mitogen.
```

Comment:
Can we enable this and require Ansible 2.7 for now?

Reply:
Mostly I just didn't want to bloat this PR with all the Mitogen stuff, so I left it out.